.docker.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -x
+
+env
+storage-ng-travis-cpp && bash <(curl -s https://codecov.io/bash)
```

Comment:
that is exactly reason why we use travis and docker inside. There are no permissions there, nothing to attack. That is why I use it. Even if code gets complete root permission, penetrate docker and even penetrate KVM to control server. It will be travis server and not our.